R/dashboard.Rmd

```diff
@@ -1,28 +1,414 @@
----
-title: "Untitled"
-output: html_document
----
-
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
-```
-
-## R Markdown
-
-This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
-
-When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
-
-```{r cars}
-summary(cars)
-```
-
-## Including Plots
-
-You can also embed plots, for example:
-
-```{r pressure, echo=FALSE}
-plot(pressure)
-```
-
-Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
+---
+title: "VNSO Trade Dashboard"
+output: 
+  flexdashboard::flex_dashboard:
+    orientation: rows
+    vertical_layout: scroll
+    runtime: shiny
+---
+
+```{r setup, include=FALSE}
+# Load libraries
+library(flexdashboard)
+library(tidyverse)
+library(DT)
+library(dplyr)
+library(gganimate)
+library(plotly)
+library(knitr)
+library(shiny)
+
+# Note repository location
+repository <- dirname(getwd()) # Note that to run code inline you'll need to set session working directory to source file location using Session in toolbar above
+```
+
+```{r Load files}
+toptenexport <- read.csv(file.path(repository, "data", "secure", "tab.csv"))
+countryYearExport <- read.csv(file.path(repository, "data", "secure", "country_export.csv"))
+curtrade <- read.csv(file.path(repository, "data", "secure", "curtrade.csv"))
+exp <- read.csv(file.path(repository, "data", "secure", "country_year_export.csv"))
+imp <- read.csv(file.path(repository, "data", "secure", "country_year_import.csv"))
+services <- read.csv(file.path(repository, "data", "secure", "tblservices.csv"))
+trades <- read.csv(file.path(repository, "data", "secure", "trades.csv"))

```

Comment:
Using `file.path()` function and relative paths here - hopefully should work for you as well. Use the `Session` menu in RStudio to set working directory if you're wanting to run single code cells (outside of building dashboard.